bin-stub/phparkitect

```diff
@@ -7,28 +7,7 @@ use Arkitect\CLI\Check;
 use Arkitect\CLI\Command\Init;
 use Arkitect\CLI\Version;
 
-foreach (array(__DIR__ . '/../../../autoload.php', __DIR__ . '/../../autoload.php', __DIR__ . '/../vendor/autoload.php', __DIR__ . '/vendor/autoload.php') as $file) {
-    if (file_exists($file)) {
-        define('PHPARKITECT_COMPOSER_INSTALL', $file);
-
-        break;
-    }
-}
-
-unset($file);
-
-if (!defined('PHPARKITECT_COMPOSER_INSTALL')) {
-    fwrite(
-        STDERR,
-        'You need to set up the project dependencies using Composer:' . PHP_EOL . PHP_EOL .
-        '    composer install' . PHP_EOL . PHP_EOL .
-        'You can learn all about Composer on https://getcomposer.org/.' . PHP_EOL
-    );
-
-    die(1);
-}
-
-require PHPARKITECT_COMPOSER_INSTALL;
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
AFAIK the only thing the above code is supposed to do is setting PHPARKITECT_COMPOSER_INSTALL as a way to say the package was installed via composer 🤔

To me it can be safely removed

On the other hand, I am not totally sure about adding this line: what is it effect?

Reply:
To me seems more like:
 - try to find and autoloader (why? it's always in the same place)
 - setup a warning if `composer install` has not been executed yet
 - include the autoload

The situation is different if we are searching the autoload of the project ouside of the phar, but I don't think it's this case.

Reply:
you are totally right... let's remove it then